install/Knative-custom-install.md

```diff
@@ -178,33 +180,33 @@ files from the Knative repositories:
+| **knative/serving**                              |                                                                                                       |                                                                   |
+| **[`serving.yaml`][1.1]_†_**                     | Installs the  Serving component.                                                                      |                                                                   |
+| **[`monitoring.yaml`][1.2]_†_**                  | Installs the [ELK stack][2], [Prometheus][2.1], [Grafana][2.2], and [Zipkin][2.3]**\***               | Serving component                                                 |
+| [`monitoring-logs-elasticsearch.yaml`][1.3]      | Installs only the [ELK stack][2]**\***                                                                | Serving component                                                 |
+| [`monitoring-metrics-prometheus.yaml`][1.4]      | Installs only [Prometheus][2.1]**\***                                                                 | Serving component                                                 |
+| [`monitoring-tracing-zipkin.yaml`][1.5]          | Installs only [Zipkin][2.3].**\***                                                                    | Serving component, ELK stack (monitoring-logs-elasticsearch.yaml) |
+| [`monitoring-tracing-zipkin-in-mem.yaml`][1.6]   | Installs only [Zipkin in-memory][2.3]**\***                                                           | Serving component                                                 |
+| **knative/build**                                |                                                                                                       |                                                                   |
+| **[`release.yaml`][3.1]_†_**                     | Installs the Build component.                                                                         |                                                                   |
+| **knative/eventing**                             |                                                                                                       |                                                                   |
+| **[`release.yaml`][4.1]_†_**                     | Installs the Eventing component. Includes the in-memory channel provisioner.                          | Serving component                                                 |
+| [`eventing.yaml`][4.2]                           | Installs the Eventing component. Does not include the in-memory channel provisioner.                  | Serving component                                                 |
+| [`in-memory-channel.yaml`][4.3]                  | Installs only the in-memory channel provisioner.                                                      | Eventing component                                                |
```

Comment:
Nit: the dependency for these channel install files and any of the event sources below is both Eventing and Serving 

(just another one of those unnecessary cognitive load things where we force them to read the whole eventing row to mentally map more dependencies vs. all in one place)

Reply:
I added in the additional/implied dependencies.